install-scripts/install-debian.sh

```diff
@@ -0,0 +1,40 @@
+# This installation has been tested on Debian 11 XFCE
+
+# Install curl in order to install Rust and Cargo
+# Install make, necessary for installing python 3.9 with pyenv
+
+sudo apt install -y curl 
+                    \ make
+
+# Install Rust and Cargo
+curl https://sh.rustup.rs -sSf | sh
+
+# Make sure Rust has been installed correctly
+rustc --version
+
+# Install pyenv dependencies
+sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev
+
+# Install pyenv
+curl https://pyenv.run | bash
```

Comment:
Same as with rust: do we have to source some files to have `pyenv` available in our PATH?